CHANGES.md

```diff
@@ -134,6 +134,7 @@ and the first release covered by our new
 
 ### Integrations
 
+- Move test to disable plugin in vim/neovim, which speeds up loading (#2896)
```

Comment:
This is the changelog for the already released 22.1.0, move your entry to the unreleased section at line 36

Reply:
Sorry!